numpy/f2py/crackfortran.py

```diff
@@ -2271,9 +2265,8 @@ def analyzebody(block, args, tab=''):
             as_ = args
         b = postcrack(b, as_, tab=tab + '\t')
         if b['block'] in ['interface', 'abstract interface'] and \
-           not b['body'] and not b.get('implementedby'):
-            if 'f2pyenhancements' not in b:
-                continue
+           not b['body'] and not b.get('implementedby') and 'f2pyenhancements' not in b:
```

Comment:
Neither is very readable here, I'd put more of the bits in separate lines.

Reply:
Agreed, but if numpy starts using a formatter some day, the formatter will unfold the line.